app/forms/custom_form_helpers.rb

```diff
@@ -0,0 +1,24 @@
+module CustomFormHelpers
```

Comment:
This is a rails pattern I've used in the past. I'll update to something more meaningful.